app/src/main/java/com/example/scoutconcordia/DataStructures/Graph.java

```diff
@@ -1,4 +1,4 @@
-package com.example.scoutconcordia;
+package DataStructures;
```

Comment:
should be : package com.example.scoutconcordia.DataStructures;

Reply:
When a class is in the same package Java imports the items automatically unless you want Node to be a separate class though it is only intended for use with the Graph. The naming has been fixed.